diff --git a/node_modules/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt b/node_modules/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt
index 999a22c..36b109b 100644
--- a/node_modules/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt
+++ b/node_modules/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt
@@ -135,6 +135,7 @@ class ExpoCameraView(
 
   private var previewView = PreviewView(context).apply {
     elevation = 0f
+    implementationMode = PreviewView.ImplementationMode.COMPATIBLE
   }
   private val scope = CoroutineScope(
     Dispatchers.Main + SupervisorJob() + CoroutineExceptionHandler { _, throwable ->
